documentation/src/main/asciidoc/mapper-orm-indexing-automatic.asciidoc

```diff
@@ -37,7 +37,8 @@ This is because queries are executed on the database side,
 One workaround is to <<mapper-orm-indexing-explicit,explicitly reindex>> after you run such queries.
 Indexing happens after transactions are committed::
 Indexes are not updated immediately, but only after transactions are successfully committed.
-That way, if a transaction is rolled back, the indexes will be left in a state consistent with the database.
+That way, if a transaction is rolled back, the indexes will be left in a state consistent with the database,
+discarding all the relative changes and releasing the relative resources.
```

Comment:
I'm not sure "relative" can be used this way in this context.

Maybe: `discarding all the index changes that were planned during the transaction.`?

Reply:
I translated literally from the Italian.. :P. Maybe `something that is relative to something else` doesn't have the same meaning in English... So let's change it.